packages/wallet/wdk/src/sequence/sessions.ts

```diff
@@ -327,12 +342,12 @@ export class Sessions implements SessionsInterface {
   async addExplicitSession(
     walletAddress: Address.Address,
     sessionAddress: Address.Address,
```

Comment:
sessionAddress is an argument for addExplicitSession but its also a property of the explicitSession.